app/views/invoices/show.html.erb

```diff
@@ -7,4 +7,24 @@
 <section id="invoice-customer-info">
   <h2>Customer:</h2>
   <p><%= @merchant_invoice.customer.name %></p>
-</section>
\ No newline at end of file
+</section>
+
+<section id="invoice-items-info">
+  <h3>Items on this Invoice: </h3>
+  <table>
```

Comment:
Yes! Wonderful use of a table here so we get this displayed nicely on the show page!